tests/integration/it_locations_test.py

```diff
@@ -37,16 +37,33 @@ def test_create(client):
 
 
 def test_edit(client):
-    assert client.get("/locations/edit/1").status_code == HTTPStatus.OK
+    location_old = factories.LocationFactory()
+    location_data = {
```

Comment:
@vladarefiev can we name these two ``original`` and ``edited`` rather than ``old`` and ``data``?

Reply:
@amihaiemil ok